pcsx2/gui/AppGameDatabase.cpp

```diff
@@ -148,42 +39,38 @@ AppGameDatabase& AppGameDatabase::LoadFromFile(const wxString& _file, const wxSt
 
 		//wxDirName dir = InstallFolder;
 		wxDirName dir = (wxDirName)wxFileName(wxStandardPaths::Get().GetExecutablePath()).GetPath();
-		file = ( dir + file ).GetFullPath();
+		file = (dir + file).GetFullPath();
 	}
-	
-	
+
+
 	if (!wxFileExists(file))
 	{
 		Console.Error(L"(GameDB) Database Not Found! [%s]", WX_STR(file));
 		return *this;
 	}
 
-	wxFFileInputStream reader( file );
+	u64 qpc_Start = GetCPUTicks();
 
-	if (!reader.IsOk())
+	if (!this->initDatabase(std::string(file)))
```

Comment:
This also may be Unicode unfriendly on Windows if a full absolute path is passed. If it's only a relative path within PCSX2, disregard.